caseenum/src/main/scala/macro.scala

```diff
@@ -116,7 +116,7 @@ object IndexedEnumMacro {
       }
       val typeAliasTree :: memberTrees = body
       val members = memberTrees.map {
-        case q"object $memberName { $statement }" =>
+        case Apply(Ident(memberName: TermName), List(Literal(statement))) =>
```

Comment:
Just a heads-up. I think this changes semantics: 
`$statement` matches any expression, while `Literal(statement)`
But, it may be definitely sensible to restrict the value to just literals.